app/src/test/java/at/aau/edu/wizards/gameModel/GameModelPlayerUnitTest.kt

```diff
@@ -93,4 +94,6 @@ class GameModelPlayerUnitTest {
         assertEquals(-100, player.scores[4])
         assertEquals(100, player.scores[5])
     }
+
+
```

Comment:
One more file to check, every change... But why adding two empty lines here? One line would be correct, I see that.